lib/plugins/aws/package/compile/events/apiGateway/lib/validate.js

```diff
@@ -395,7 +395,7 @@ module.exports = {
         cors.methods.push(http.method.toUpperCase());
       }
       if (_.has(cors, 'maxAge')) {
-        if (!_.isInteger(cors.maxAge) || cors.maxAge < 1) {
+        if (!Number(cors.maxAge) || cors.maxAge < 1) {
```

Comment:
Note that `Number` does coercion to number, so this is useless in this condition. Let's just keep it as `if (cors.maxAge < 1)`